app/api/dao/mentorship_relation.py

```diff
@@ -187,6 +191,20 @@ def accept_request(user_id, request_id):
         request.state = MentorshipRelationState.ACCEPTED
         request.save_to_db()
 
+        # Send mail
+        from run import application
+
+        template = render_template('email_mentorship_accepted.html', user_name=user.name)
+        email_user = UserModel.find_by_id(request.action_user_id)
+
+        msg = Message(
+            "Mentorship System - Mentorship Relation accepted!",
+            recipients=[email_user.email],
+            html=template,
+            sender=application.config['MAIL_DEFAULT_SENDER']
+        )
+        mail.send(msg)
```

Comment:
Hi, can you please use the suitable replacement for this from the `email_utils.py` ?